public/app/plugins/datasource/tempo/traceql/autocomplete.ts

```diff
@@ -81,11 +83,8 @@ export class CompletionProvider implements monacoTypes.languages.CompletionItemP
     });
   }
 
-  /**
-   * We expect the tags list data directly from the request and assign it an empty set here.
-   */
-  setTags(tags: string[]) {
-    tags.forEach((t) => (this.tags[t] = new Set<string>()));
```

Comment:
Not sure why we were setting each tag to an empty set before. Guessing it's from an older implementation that has since been updated